addon/models/osf-model.js

```diff
@@ -20,20 +20,25 @@ export default DS.Model.extend(HasManyQuery.ModelMixin, {
         this._super(...arguments);
         this.set('_dirtyRelationships', Ember.Object.create({}));
     },
+    resolveRelationship(rel) {
```

Comment:
This seems to be a custom method of our own devising?

For the future, a docstring would be helpful for all newly added methods. We're accessing some pretty deep internals of ember data, and documenting the what and why would do a lot to help maintainability.